pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -151,6 +151,42 @@ func (pl *VolumeBinding) isSchedulableAfterCSINodeChange(logger klog.Logger, pod
 	return framework.QueueSkip, nil
 }
 
+func (pl *VolumeBinding) isSchedulableAfterCSIStorageCapacityChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
+	oldCap, newCap, err := util.As[*storagev1.CSIStorageCapacity](oldObj, newObj)
+	if err != nil {
+		return framework.Queue, err
+	}
+
+	logger = klog.LoggerWithValues(
+		logger,
+		"Pod", klog.KObj(pod),
+		"CSIStorageCapacity", klog.KObj(newCap),
+	)
+
+	if oldCap == nil {
+		logger.V(5).Info("A new CSIStorageCapacity was created, which could make a Pod schedulable")
+		return framework.Queue, nil
+	}
+
+	if newCap.Capacity != oldCap.Capacity {
```

Comment:
We can do the same with capacity too, right?

Reply:
I have made the necessary changes to address this. Thank you for pointing it out.

https://github.com/kubernetes/kubernetes/pull/124961/commits/e6a19136ad3027f209225fbd347e5424eeb04077